> ava.lang.AssertionError: Found features violating visibility conditions: 
> Found issues with Feature: [io.openliberty.faces-4.1]   Edition: [core]   Kind: [beta]
>      It provisions less visible features: 
>      Feature: [io.openliberty.facesProvider-4.1.0.MyFaces]   Edition: [full]   Kind: [noship]
>

--- a/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.facesProvider-4.1.0.MyFaces.feature
+++ b/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.facesProvider-4.1.0.MyFaces.feature
@@ -4,6 +4,6 @@
 symbolicName=io.openliberty.facesProvider-4.1.0.MyFaces
 singleton=true
 visibility=private
-kind=noship
-edition=full
+kind=beta
+edition=core
 WLP-Activation-Type: parallel

--- a/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.faces-4.1.feature
+++ b/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.faces-4.1.feature
@@ -1,9 +1,8 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.jakarta.faces-4.1
 singleton=true
--features=com.ibm.websphere.appserver.eeCompatible-11.0, \
-  io.openliberty.noShip-1.0
+-features=com.ibm.websphere.appserver.eeCompatible-11.0
 -bundles=io.openliberty.jakarta.faces.4.1; location:="dev/api/spec/,lib/"; mavenCoordinates="org.apache.myfaces.core:myfaces-api:4.1.0"
-kind=noship
-edition=full
+kind=beta
+edition=core
 WLP-Activation-Type: parallel

--- a/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.websocket-2.2.feature
+++ b/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.websocket-2.2.feature
@@ -1,11 +1,10 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.jakarta.websocket-2.2
 singleton=true
--features=com.ibm.websphere.appserver.eeCompatible-11.0, \
-  io.openliberty.noShip-1.0
+-features=com.ibm.websphere.appserver.eeCompatible-11.0
 -bundles=\
   io.openliberty.jakarta.websocket.2.2; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.websocket:jakarta.websocket-api:2.2.0", \
   io.openliberty.jakarta.websocket.client.2.2; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.websocket:jakarta.websocket-client-api:2.2.0"
-kind=noship
-edition=full
+kind=beta
+edition=core
 WLP-Activation-Type: parallel

--- a/dev/com.ibm.websphere.appserver.features/visibility/public/faces-4.1/io.openliberty.faces-4.1.feature
+++ b/dev/com.ibm.websphere.appserver.features/visibility/public/faces-4.1/io.openliberty.faces-4.1.feature
@@ -50,8 +50,8 @@ Subsystem-Name: Jakarta Faces 4.1
  com.ibm.ws.cdi.interfaces.jakarta, \
  com.ibm.ws.cdi.2.0.jsf.jakarta, \
  io.openliberty.faces.4.1.thirdparty; location:="dev/api/third-party/"
-kind=noship
-edition=full
+kind=beta
+edition=core
 WLP-Activation-Type: parallel
 WLP-InstantOn-Enabled: true
 WLP-Platform: jakartaee-11.0

--- a/dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/singleton_expected.xml
+++ b/dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/singleton_expected.xml
@@ -12137,7 +12137,6 @@
         </input>
         <output>
             <resolved>com.ibm.websphere.appserver.eeCompatible-11.0</resolved>
-            <resolved>noShip-1.0</resolved>
             <resolved>io.openliberty.jakarta.websocket-2.2</resolved>
             <resolved>com.ibm.websphere.appserver.appLifecycle-1.0</resolved>
             <resolved>com.ibm.websphere.appserver.artifact-1.0</resolved>

--- a/dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/singleton_expected_WL.xml
+++ b/dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/singleton_expected_WL.xml
@@ -8902,7 +8902,6 @@
     </input>
     <output>
         <resolved>com.ibm.websphere.appserver.eeCompatible-11.0</resolved>
-        <resolved>noShip-1.0</resolved>
         <resolved>io.openliberty.jakarta.websocket-2.2</resolved>
         <resolved>com.ibm.websphere.appserver.appLifecycle-1.0</resolved>
         <resolved>com.ibm.websphere.appserver.artifact-1.0</resolved>

